contracts/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# @chainlink/contracts CHANGELOG.md
+
+## 0.2.0 (unreleased)
+
+Features:
+
+- Contracts specific changelog
+- `KeeperCompatibleInterface` contracts
+- v0.8 Consumable contracts (`ChainlinkClient`, `VRFConsumerBase` and Aggregator interfaces)
+- Multi-word response in v0.7 and v0.8 `ChainlinkClient` contracts
+- Tests converted to Hardhat
```

Comment:
Are there any dependency changes that people need to know about? Breaking changes? Will we still ship ethers and truffle types? Change in Ethers version types? If yes to any of those, or anything else that downstream consumers will notice, we should definitely note it here.

Reply:
Added some more, I think that's everything.